docs/Demo.md

```diff
@@ -12,37 +8,137 @@
 
 In this demo, you will get a taste of how Pelorus captures a change going through the application's delivery cycle.
 
-1. Initializing Pelorus sets the baseline by looking at existing stored data
-2. Create a new commit
-3. Watch as the metrics and trends change as new versions roll out
+* Install and configure Pelorus
+* Install a sample application that Pelorus will measure
+* Set a baseline of data for Pelorus measurements
+* Create a new commits, and Github issues
+* Watch as the metrics and trends change as new versions roll out
 
-Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
+Understand that Pelorus should be used as a conversation tool to read the trends in metrics and react by making informed investments in the software delivery process.
 
-## Prerequisites
+--------------------------
 
-* Pelorus is installed and running
-  * Follow the steps outlined in the [todolist install guide](../samples/todolist.md#pelorus-install-steps)
+# Lead Time for Change and Deployment Frequency
```

Comment:
I think it's worth mentioning here something like:

Pelorus allows to measure four metrics (https://pelorus.readthedocs.io/en/latest/dashboards/SoftwareDeliveryPerformance/). For this demo two of them will be presented.

Reply:
Under the DORA img.. I'm adding.
> **Note:** More information about the four key DORA metrics can be found at the [Software Delivery Performance section](Dashboards.md) 

I think that will cover this.